n2survey/lime.py

```diff
@@ -0,0 +1,436 @@
+"""Lime Survey related helper functions
+
+The module contains helper functions for parsing
+lime survey data such as *.xml structure files
+"""
+
+import os
+import re
+from typing import Optional
+from warnings import warn
+
+from bs4 import BeautifulSoup
+from bs4.element import Tag
+
+__all__ = [
+    "read_lime_questionnaire_structure",
+]
+
+
+def _get_clean_string(tag: Tag) -> str:
+    """Clear a text in XML from HTML tags and line breaks"""
+    # Remove HTML tags and line breaks
+    clean_string = (
+        " ".join(BeautifulSoup(tag.text, "html.parser").stripped_strings)
+        .replace("\n", " ")
+        .replace("\t", " ")
+        .replace("\xa0", " ")
+        .strip()
```

Comment:
oh, stripped_string is really a good choice, can I ask why there need another strip() func?